### ISSUE TITLE
Remove FSL dependency for IBMAs if possible

### ISSUE THREAD

**Issue by tsalo** [neurostuff/NiMARE#108]
It would be preferable to limit NiMARE to Python dependencies, but we currently use FSL for the fixed and mixed effects GLM IBMAs. If possible, it would be good to find pure Python tools that we could use for this. Nistats may have the means to do this.

**Comment by tsalo** [comment]
After a discussion with @nicholst, it seems like we can do this using one of a few methods for calculating between-study variance, including the HO estimator described [here](http://doi.org/10.1002/jrsm.1164) or the HKSJ method described [here](http://doi.org/10.1186/1471-2288-14-25). I'll need to actually read the papers in order to figure out how to implement them though.

**Comment by nicholst** [comment]
TL;DR the journal papers:

• HO uses a methods of moments estimator to estimate sigma_Btw, a simple
compilation of max(0, S^2 - \bar{sigma^2_{Wtn}}), where S^2 is usual/naive
sample variance of the K studies’ point estimates (ie COPEs), and
\bar{sigma^2_{Wtn}}) is the sample mean of the K studies’ estimator
variances (stderr^2 or VarCOPE). With the each study weighted by something
like 1/(sigma^2_{k,Wtn}+sigma_Btw) (I’m hedging only b/c I forget if this
is correct as is or 1/sqrt()), the variance of the meta estimator is a
simple function of the weights.

• HKSJ uses some estimate of sigma_Btw, say, that above, & computes weights
and the meta estimator in the same way, but computes the variance
differently: it uses the sample variance of the K *weighted* estimates. It
is very similar in spirit to the method Camille Maumet proposed that does a
one sample t on the Z scores.

-- 
_________________________________________________________
Thomas Nichols, PhD
Professor of Neuroimaging Statistics
Nuffield Department of Population Health | University of Oxford
Big Data Institute | Li Ka Shing Centre for Health Information and Discovery
Old Road Campus | Headington | Oxford | OX3 7LF | United Kingdom
T: +44 1865 743590 | E: thomas.nichols@bdi.ox.ac.uk
W: http://nisox.org | http://www.bdi.ox.ac.uk

**Comment by tsalo** [comment]
This will be handled in the same PR than handles #196.
